components/Pages/Project/Review/CardReviewSummary.tsx

```diff
@@ -0,0 +1,279 @@
+  const { isConnected, address, chainId } = useAccount();
+
+  const enum IntervalMessage {
+    PER_DAY = "per day",
+    PER_WEEK = "per week",
+    PER_MONTH = "per month",
+    PER_YEAR = "per year",
+  }
+
+  interface StarRatingDataProps {
+    stars: number;
+    count: number;
+    percentage: number;
+  }
+
+  // Grab all recent badges and save on state
+  const handleStoryBadges = async () => {
+    const badgeIds = await getBadgeIds(SCORER_ID);
+    const badges = badgeIds && (await Promise.all(badgeIds.map((id) => getBadge(id))));
+    setActiveBadgeIds(badgeIds);
+    setActiveBadges(badges);
+  };
+
+  const handleReviewButton = () => {
+    if (!isConnected && openConnectModal) {
+      openConnectModal();
+    } else {
+      if (chainId != arbitrum.id) {
+        switchChain({ chainId: arbitrum.id });
+        toast.error("Must connect to Arbitrum to review");
+      } else {
+        setIsOpenReview(ReviewMode.WRITE);
+        handleStoryBadges();
+      }
+    }
+  };
+
+  const score = stories?.map((grantStorie: GrantStory) => grantStorie.averageScore) || [];
+
+  /**
+   * Calculates the average review score and updates the state with the result.
+   *
+   * @param score - An array of numbers representing individual review scores.
+   * @param grantStoriesLength - The total number of grant stories.
+   */
+  const getAverageReview = (score: number[], grantStoriesLength: number) => {
+    const averageScoreSum = score.reduce(
+      (accumulator, current) => Number(accumulator) + Number(current),
+      0,
+    );
+
+    setAverageScoreReview(
+      Number(averageScoreSum) / 10 ** SCORER_DECIMALS / Number(grantStoriesLength),
+    );
+  };
+
+  const grantIdFromQueryParam = searchParams?.get("grantId");
+
+  useEffect(() => {
+    if (stories) {
+      getAverageReview(score, stories.length);
+      const timestamps = stories.map((story: GrantStory) => Number(story.timestamp));
+      getScoreRatingFilteredReviews();
+      const timestampNow = Math.floor(Date.now() / 1000); /* Get the current timestamp in seconds */
+      getTimestampInterval(timestamps, timestampNow);
+    } else {
+      setTimestampInterval(null);
+      setIntervalMessage(undefined);
+    }
+  }, [stories, grantIdFromQueryParam]);
+
+  const getTimestampInterval = (timestamps: number[], timestampNow: number) => {
+    if (!timestamps.length) return;
+    if (!stories.length) return;
+
+    const timestampStart = Math.min(...timestamps);
+    const totalInterval = timestampNow - timestampStart;
+
+    const grantStoriesLength = timestamps.length;
+    const intervalPerReview = totalInterval / grantStoriesLength;
+
+    setTimestampInterval(intervalPerReview);
+    setIntervalMessage(getIntervalMessage(intervalPerReview));
+  };
+
+  const getIntervalMessage = (interval: number): string => {
+    const days = interval / (60 * 60 * 24);
+
+    if (days <= 1) {
+      return `Typically reviewed 1 time ${IntervalMessage.PER_DAY}`;
```

Comment:
```suggestion
      return `Typically reviewed ${the number of reviews done so far} time${number of reviews done so far > 1 ? "s" : ""} ${IntervalMessage.PER_DAY}`;
```

Please put the correct constants inside the ${} I put